TrueCraft.Client/Rendering/Blocks/SlabRenderer.cs

```diff
@@ -0,0 +1,175 @@
+﻿using System;
+using TrueCraft.Core.Logic.Blocks;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using TrueCraft.API.Logic;
+
+namespace TrueCraft.Client.Rendering.Blocks
+{
+    public class SlabRenderer : BlockRenderer
+    {
+        static SlabRenderer()
+        {
+            BlockRenderer.RegisterRenderer(StoneSlabBlock.BlockID, new SlabRenderer());
+            BlockRenderer.RegisterRenderer(DoubleStoneSlabBlock.BlockID, new SlabRenderer());
+        }
+
+        public enum SlabType : byte
+        {
+            Stone = 0,
+            Sandstone = 1,
+            Wood =2,
+            Cobblestone = 3
+        }
```

Comment:
Type enums go in the class they belong to - in this case, let's rename `StoneSlabBlock` to `SlabBlock`, and make `DoubleStoneSlabBlock` into `DoubleSlabBlock` and have it inherit from `SlabBlock`.